source/_patterns/00-atoms/forms/text-field.yaml

```diff
@@ -17,9 +17,40 @@ schema:
       type: string
       enum:
         - valid
-        - error
-    message:
-      type: string
+        - invalid
+    isInvalid:
+      type: boolean
+      enum:
+        - true
+    messageGroup:
+      type: object
+      properties:
+        errorText:
+          type: string
+          minLength: 1
+        infoText:
+          type: string
+          minLength: 1
+        id:
+          type: string
+          minLength: 1
+      oneOf:
+        - required:
+          - id
+          - errorText
+          not:
+            required:
+              - infoText
+        - required:
+          - id
+          - infoText
+          not:
+            required:
+              - errorText
+        - required:
+          - id
+          - errorText
+          - infoText
```

Comment:
🙃 

Might be clear to make `id` required and just have `minProperties: 2`. (Technically that means _any_ 2 properties, but meh.)